pyqg/tests/test_xarray_output.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import print_function
+import numpy as np
+import pyqg
+import xarray as xr
+
+def test_xarray():
+    m = pyqg.QGModel(1)
+    ds = m.to_dataset()
+    
+    assert type(ds) == xr.Dataset
```

Comment:
Can we also assert that certain variables are present. E.g.
```python
expected_vars = ['q', 'phi']  # etc
for v in expected_vars:
    assert v in ds
```

Reply:
I added this assertion with a few variables I expect to always be present.